STARTERKIT/js/script.js

```diff
@@ -10,9 +10,9 @@
 // wrapping it with an "anonymous closure". See:
 // - https://drupal.org/node/1446420
 // - http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
-(function (Drupal) {
+'use strict';
```

Comment:
This hunk is wrong, please revert
http://cgit.drupalcode.org/drupal/tree/core/modules/big_pipe/js/big_pipe.es6.js#n6